week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/types/price.type.ts

```diff
@@ -0,0 +1,4 @@
+export type Price = {
+  value: number;
+  currency: 'PLN' | 'USD' | 'EUR' | 'GBP';
+};
```

Comment:
Generalnie wydzielenie `types` to dobry pomysł (odnosząc się do komentarza Twojego w PR). 
Po moim `CR` z zadania `1` wiesz już że ja ten katalog bym nazwał `models` :) 

Dodatkowo - za równo o `price` jak i o `item` można powiedzieć że są "application wide".
Wiec obydwa te typy mogą być `interfaces` i nazwane jako `*.model.ts`